packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -672,22 +696,103 @@ export class PluginLedgerConnectorBesu
       throw new CodedError(msg, E_KEYCHAIN_NOT_FOUND);
     }
 
-    const pem: string = await keychain.get(keychainRef);
+    if (nodeHostsProvider === NodeHostsProviderType.ConsortiumPlugin) {
+      const consortium = this.pluginRegistry.plugins.find(
+        (plugin) => plugin.getInstanceId() == consortiumId,
+      ) as PluginConsortiumManual;
 
-    const pkRaw = converter.privateKeyAs(pem, KeyFormat.PEM, KeyFormat.Raw);
+      if (consortium === undefined) {
+        throw new Error(`${fnTag} Consortium plugin not found`);
+      }
+
+      const keyPairPem = consortium.options.keyPairPem;
+      const totalNodes = consortium.getNodeCount();
+
+      //Get consortium nodes
+      const nodesAll = consortium.options.consortiumRepo.allNodes;
```

Comment:
@AzaharaC Please use the get consortium jws endpoint of the consortium plugin to obtain the list of nodes.